frontend/uddhar-frontend/src/coordinator/components/TeamAssignment/TableWithPagination.jsx

```
@@ -0,0 +1,265 @@
+      )}
+      {/* Pagination */}
+      <div className="flex flex-col sm:flex-row justify-between items-center mt-4 gap-4">
+        <div className="text-sm text-gray-600">
+          Showing {indexOfFirstItem + 1} to{" "}
+          {Math.min(indexOfLastItem, filteredData.length)} of{" "}
+          {filteredData.length} entries
+        </div>
+
+        <div className="flex gap-2">
+          {(() => {
+            let startPage, endPage;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)